BraveRewardsUI/Wallet/WalletViewController.swift

```diff
@@ -201,6 +205,20 @@ class WalletViewController: UIViewController, RewardsSummaryProtocol {
         })
       }
     }
+    
+    publisherSummaryView.autoContributeChanged = { [weak self] enabled in
+      guard let self = self, let host = self.state.url.host else { return }
+      
+      // setting publisher exclusion to .included didn't seem to work, using .default instead.
+      let state: PublisherExclude = enabled ? .default : .excluded
+      self.state.ledger.updatePublisherExclusionState(withId: host, state: state)
+      
+      if !enabled {
+        // Disabled auto-contribute resets attention value.
+        attentionView.valueLabel.text = "0%"
```

Comment:
This is _probably_ fine to imply, but we might want to instead attempt to retrieve the activity (even though it'll just create a new empty one I think) and set the value based on that.